tsec-http4s/src/test/scala/tsec/authentication/AuthenticatorSpec.scala

```diff
@@ -102,7 +103,9 @@ abstract class AuthenticatorSpec extends TestSpec with MustMatchers with Propert
     def dAll(): Unit = storageMap.clear()
   }
 
-  def AuthenticatorTest[A](title: String, authSpec: AuthSpecTester[A]) = {
+  def AuthenticatorTest[A](title: String, authSpec: AuthSpecTester[A])(
+      implicit AE: ApplicativeError[OptionT[IO, ?], Throwable]
```

Comment:
Same reason here.

Reply:
Is this really not implicitly offered :/

Reply:
Yeah, it's sad.